### PR TITLE
Add support to log request content and response content

### PR DESCRIPTION
**Pull request by andre-djsystem**

### CODE REVIEW

File: src/Horse.Logger.Manager.pas [hunk 1]
```diff
@@ -93,12 +93,17 @@ procedure DefaultHorseCallback(AReq: THorseRequest; ARes: THorseResponse; ANext:
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('request_content_encoding', THorseLoggerManager.ValidateValue(AReq.RawWebRequest.ContentEncoding));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('request_content_type', THorseLoggerManager.ValidateValue(AReq.RawWebRequest.ContentType));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('request_content_length', THorseLoggerManager.ValidateValue(AReq.RawWebRequest.ContentLength.ToString));
+      LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('request_content', THorseLoggerManager.ValidateValue(AReq.RawWebRequest.Content));
```
Comment (viniciussanchez) [inline src/Horse.Logger.Manager.pas:96]: Add "request_content" and "response_content" attribute in providers documentation

File: src/Horse.Logger.Manager.pas [hunk 2]
```diff
@@ -93,12 +93,17 @@ procedure DefaultHorseCallback(AReq: THorseRequest; ARes: THorseResponse; ANext:
...
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_server', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.Server));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_allow', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.Allow));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_location', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.Location));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_content_encoding', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.ContentEncoding));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_content_type', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.ContentType));
       LLog.{$IFDEF FPC}Add{$ELSE}AddPair{$ENDIF}('response_content_length', THorseLoggerManager.ValidateValue(ARes.RawWebResponse.ContentLength.ToString));
+      if (not Assigned(ARes.Content)) or (not ARes.Content.InheritsFrom({$IF DEFINED(FPC)}TJsonData{$ELSE}TJSONValue{$ENDIF})) then
```
Comment (viniciussanchez) [inline src/Horse.Logger.Manager.pas:103]: Couldn't treat everything as a string?I believe that just put middleware before or after Jhonson middleware. jhonson Middleware will convert the JSON to string now. And the content will be string after it goes through the jhonson middleware.

Comment (andre-djsystem) [reply src/Horse.Logger.Manager.pas:103]: When used Res.Content(LPong); doesn't work  like string, but if use Res.Send(LPong.AsJson) then works fine.

File: src/Horse.Logger.Manager.pas (hunk 1 above)
Comment (andre-djsystem) [reply src/Horse.Logger.Manager.pas:96]: Done

**Pull request merged**